docs/source/examples/cirq-ibmq-backends.md

```diff
@@ -34,7 +34,7 @@ from mitiq import zne
 from mitiq.interface.mitiq_qiskit.qiskit_utils import initialized_depolarizing_noise
 
 
-USE_REAL_HARDWARE = False
+USE_REAL_HARDWARE = True
```

Comment:
I don't think we want the docs to try to use real hardware, do we?

Reply:
Actually this is causing a failure on RTD: https://mitiq--1847.org.readthedocs.build/en/1847/examples/cirq-ibmq-backends.html

Reply:
Oh! Sorry about that. I had switched them to real hardware to test some of the changes and make sure IBMProvider worked. Latest merge fixes this.